tonic_validate/utils/telemetry.py

```diff
@@ -76,16 +78,25 @@ def log_run(self, num_of_questions: int, metrics: List[str]):
             The number of questions asked
         metrics: List[str]
             The metrics that were used to evaluate the run
+        run_time: float
+            The time taken to evaluate the run
         """
         if self.config.TONIC_VALIDATE_DO_NOT_TRACK:
             return
+        try:
+            sdk_version = pkg_resources.get_distribution("tonic-validate")
```

Comment:
this is type DistInfoDistribution, not string.  It has a str property called version though.

Reply:
yeah, just confirmed.  Current code throws a swalled exception because DistInfoDistrubtion (return type from get_distribution call) is not serializable to JSON.

```
            sdk_version = pkg_resources.get_distribution("tonic-validate").version
```